### ISSUE TITLE
After recent changes on Akamai create/refresh/download tokens fails when decoding JWT

### ISSUE THREAD

**Issue by ciandt-crodrigues** [akamai/cli-edgeworkers#155]
Commands like 
`akamai edgekv create token token1 --overwrite --staging=allow --production=allow --ewids=all --namespace=default+rwd --expiry=9999-12-31` 
or
`akamai edgekv download token token1` 

fails with error "Error while trying to decode the JWT token"

the reason is because this was not updated to reflect the recent changes on the token described [here](https://techdocs.akamai.com/edgekv/docs/generate-and-retrieve-edgekv-access-tokens#troubleshoot-access-tokens)

It is a real problem for me because my pipeline uses download to retrieve the token to build new edgeworker versions (because we should not version tokens on our version control)

the issue is probably [here](https://github.com/akamai/cli-edgeworkers/blob/master/src/edgekv/ekv-handler.ts#L811):

Please Release a fix for this ASAP!

**Comment by paul-bartlett** [comment]
Working on a release that should address this right now, should be out later today or early tomorrow

**Comment by paul-bartlett** [comment]
Release just went out, can you update and try again?

**Comment by ciandt-crodrigues** [comment]
I can confirm it is working now.
But no new version? I found strange having it with the same version as before but the code is different

**Comment by paul-bartlett** [comment]
Did you run `akamai edgekv --version`? I bumped to 1.9.0 for this release

**Comment by ciandt-crodrigues** [comment]
1.9.0 was released already.
on this commit, I think: https://github.com/akamai/cli-edgeworkers/commit/acda6b530748b47cc2bdde10e1b3f8fbe1bd6b78

https://github.com/akamai/cli-edgeworkers/releases/tag/v1.9.0
to bump it I think you need to update the version on the package.json

**Comment by jos76** [comment]
Has the fix for this been released? I'm on version `1.9.0` and I'm still seeing the following error:
> Error while trying to decode the JWT token

while trying to run:
```akamai edgekv download token token1```

**Comment by paul-bartlett** [comment]
Yup the fix was released, I can look into this if you send me an account ID (or maybe show output for `list tokens`)

**Comment by jos76** [comment]
I was able to get the token using the API instead of the CLI
